From df142f7508a9e8aea6fdedca7b0a0a7c16b872e8 Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Sun, 2 Jun 2019 02:49:10 +0200
Subject: [PATCH] modules: update OpenWrt

6761961919 openssl: update to 1.0.2s
9591155737 kernel: Fix arc kernel 4.14 build
dc1b578a4c curl: Fix multiple security problems
40ed8389ef mbedtls: update to version 2.16.1
7e07320dc4 kernel: bump 4.14 to 4.14.121
054aecdf0b kernel: bump 4.9 to 4.9.178
e9a7344550 uci: fix heap use after free (FS#2288)
4b633affff brcm63xx: drop linux,part-probe usage where possible
cfb72eed69 brcm63xx: drop own implementation of DT partitions in favour of upstream
68a5e662c2 kernel: bump 4.14 to 4.14.120
85294fc5e7 kernel: bump 4.9 to 4.9.177
152755c9a2 kernel: bump 4.14 to 4.14.119
82e4b4250d kernel: bump 4.9 to 4.9.176
cf2aa873ea ar71xx: Fix network setup for TP-Link Archer C25 v1
e6928e6b29 kernel: Fix arc kernel build
3239f56136 uboot-fritz4040: Add host flags for host compiler
b5ce5217e2 ramips: rt305x: Reduce size of a5-v11 image
d3053b1bdc kernel: bump 4.14 to 4.14.118
f053a8ce41 kernel: bump 4.9 to 4.9.175
412d80cdb7 kernel: bump 4.14 to 4.14.115
f105a9c35c kernel: bump 4.9 to 4.9.172
4685bf1d2f kernel: bump 4.14 to 4.14.114
2faceb1a39 kernel: bump 4.9 to 4.9.171
e0505cc018 Revert "uhttpd: disable concurrent requests by default"
24aefaec62 tools/pkg-config: Handle variable substitution of 'bindir' to redirect to STAGING_DIR/bin
aaa34526c4 tools/pkg-config: pass arguments at the end
3103bd54c5 kernel: bump 4.14 to 4.14.113
a5c62c933b kernel: bump 4.9 to 4.9.170
ac3b5f00e1 kernel: bump 4.14 to 4.14.112
15a70d085d kernel: bump 4.9 to 4.9.169
e6e5435c5b ar71xx: GL.iNet AR300M family: correct LED definitions
ecfe0f1cc4 ca-certificates: update to version 20190110
2d2e615dee mac80211: brcmfmac: really add early fw crash recovery
02aed76968 mac80211: brcmfmac: early work on FullMAC firmware crash recovery
1a6d7a651f lantiq: tdw89x0: Fix WLAN LED on TP-Link W8970 v1.2 (FS#2232)
6ac061f319 ar71xx: Fix IMAGE_SIZE for TP-Link Archer C7 v5
7268ebb1e4 ar71xx: Correct MAC address for WAN interface of Archer C7 v5
c7eb679047 ar71xx: Add "info" partition for TP-Link Archer C7 v5
9c4fa1bb5b ar71xx: Remove ath10k packages from archer-c7-v1 (fixes FS#1743)
6c81f5fac6 kernel: bump 4.14 to 4.14.111
aa0e6fc489 kernel: bump 4.9 to 4.9.168
dad220a00c kernel: bump 4.14 to 4.14.110
07bd5b7a81 kernel: bump 4.9 to 4.9.167
4b4de23dfb openssl: update to 1.0.2r

Build-tested: x86-64
(cherry picked from commit f8bbdd96f0fb519ee7294a9a33548ae677bc465f)
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 0549f9a7..3ed0e6f2 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing luci gluon'
 
 OPENWRT_REPO=https://git.openwrt.org/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-18.06
-OPENWRT_COMMIT=ca8b4d684b57d7eee0329eae86d68ec5581615f7
+OPENWRT_COMMIT=67619619195aa57b2d3169b659e677cd8b879e2a
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-18.06
-- 
2.17.1

